packages/common/src/utils/validateSceneName.ts

```diff
@@ -0,0 +1,28 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+export default function isValidSceneName(sceneName: string) {
+  return sceneName.length >= 3 && sceneName.length <= 64 && sceneName.indexOf('_') === -1
```

Comment:
Is it checking for valid file name? Because I think some charaters like `/`, etc are also not allowed in file names.

Ref:
https://www.npmjs.com/package/filename-reserved-regex
https://github.com/sindresorhus/filename-reserved-regex/blob/main/index.js

Reply:
Thanks a lot for pointing this out. Added these and updated the regex to include `_`
currently since we do not have common file for putting all our regex in, we can refactor all our regex used across packages to such a regex file.